src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -42,6 +42,9 @@ public class RazorView : IView
             IsPartial = isPartial;
         }
 
+        /// <inheritdoc />
+        public string Path => RazorPage.Path;
```

Comment:
We got an ok from the engineering team to use C# 6.0 features that aren't experimental. That said, seeing this is so contentious I'll just switch back to using a regular property declaration.